RUFAS/routines/manure/gas_emissions/calculator.py

```diff
@@ -10,11 +10,11 @@
 
 class GasEmissionsCalculator:
     @classmethod
-    def methane_emission_from_slurry_storage(
+    def calculate_methane_emission_from_slurry_storage(
```

Comment:
We can make this non-specific to slurry storage, as this method is currently called by the anaerobic lagoon storage files, and lagoons do not strictly hold slurry manure. I also modified the function name to match the format of the ammonia emission equation. I did not make this suggestion for all instances, I figure this change is better suited for 'replace all instances' in an IDE!

```suggestion
    def calculate_liquid_storage_methane(
```

Reply:
Sounds good!